apps/google-analytics-4/frontend/src/components/config-screen/api-access/display/DisplayServiceAccountCard.tsx

```diff
@@ -181,7 +178,7 @@ const DisplayServiceAccountCard = (props: Props) => {
       return (
         <RenderSimpleBadgeNote
           badgeLabel="Unknown Error"
-          noteMessage={'An unexpected error has occurred.'}
+          noteMessage={'Unknown error occurred. Please contact support for help.'}
```

Comment:
Cool, I think we can (should?) evolve this but it's actually a just fine start for now. Just a small copy suggestion:

> An unknown error occurred. You can try the action again in a few minutes, or contact support if the error persists.